cmd/microcloud/main_init_preseed.go

```diff
@@ -311,11 +301,6 @@ func (p *Preseed) validate(name string, bootstrap bool) error {
 		}
 	}
 
-	usingOVN := p.OVN.IPv4Gateway != "" || p.OVN.IPv6Gateway != "" || containsUplinks
```

Comment:
Is the OVN check intended to be removed here too?

Reply:
There is also another OVN check removed down below.

Reply:
Yes, there was aslso a 3 system restriction in preseed mode for OVN setup, so these were removed for that purpose. `usingOVN` is redefined elsewhere for other use cases.